src/StateStore/store.ts

```diff
@@ -151,13 +151,13 @@ class stateStore implements Store {
         this.currentCategoryPosts[imageData.post_index].thumbnail_image = imageData.image_url;
     };
 
-    setCurrentPost = (postData: Post) => {
+    setCurrentPost = (postData: Post, tagNames: string[] = [], imageUrl = '') => {
```

Comment:
Why don't you also define that `imageUrl` is a string?

Reply:
Because TypeScript can infer from the assignment that it is a string, and noted best practice is to let TypeScript infer native types most of the time (unless you find an issue with how it does it).